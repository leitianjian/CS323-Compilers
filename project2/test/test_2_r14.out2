Error type 13 at Line 10: accessing an undefined structure member
Error type 5 at Line 10: unmatching type on both sides of assignment
Error type 13 at Line 12: accessing an undefined structure member
Error type 5 at Line 12: unmatching type on both sides of assignment
