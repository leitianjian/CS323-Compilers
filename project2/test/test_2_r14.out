Error type 14 at line 10: accessing an undefined structure member
Error type 14 at line 12: accessing an undefined structure member
